_conferences/droidcon-uk-2017.md

```diff
@@ -5,4 +5,7 @@ location: London, UK
 
 date_start: 2017-10-26
 date_end:   2017-10-27
+
+ cfp_end:   2017-05-14 
```

Comment:
It seems silly to ask people to put a dummy `cfp_start`, so I think modifying `index.html` is the way to go.

If you want to add build step though, it'll be good to verify that files are added to the `_conferences` directory.